schemas/data-schema.json

```diff
@@ -124,7 +124,7 @@
         "ui:help": {"type": "string"},
         "ui:widget": {
           "type": "string",
-          "enum": ["radio", "select", "textarea", "password", "color", "updown", "range", "hidden", "file", "checkboxes"]
+          "enum": ["radio", "select", "textarea", "password", "color", "updown", "range", "hidden", "file", "checkboxes", "AutocompleteWidget"]
```

Comment:
done

Reply:
wait, that's not what I meant :) The "dj:autocomplete" schema should be at the same position as the other "dj:" or "ui:" configs. I just commited it. See https://github.com/innogames/dr-json/commit/7a465a5e0cfba12c90564e8dbba7dfe41543f47e

Reply:
Aah ok, I also thought it was strange to put it in "ui:widget" enum but I thought maybe there is some magic happening there since it was still working. 
Ok now I understand where it belongs, makes more sense :)